conan/cli/commands/list.py

```diff
@@ -108,15 +108,21 @@ def list(conan_api: ConanAPI, parser, *args):
     parser.add_argument("-g", "--graph", help="Graph json file")
     parser.add_argument("-gb", "--graph-binaries", help="Which binaries are listed", action="append")
     parser.add_argument("-gr", "--graph-recipes", help="Which recipes are listed", action="append")
+    parser.add_argument('--lru', default=None, action=OnceArgument,
+                        help="List only elements that have been recently used")
```

Comment:
I think this description should be extended, it gives no clue about what type of argument you need to add to `--lru`